src/main/java/net/coderbot/patchwork/access/ModAccessTransformations.java

```diff
@@ -3,22 +3,22 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class AccessTransformations {
-	private AccessTransformation classTransformer;
-	private Map<String, AccessTransformation> fieldTransformers;
-	private Map<String, Map<String, AccessTransformation>> methodTransformers;
+public class ModAccessTransformations {
```

Comment:
This is intended to specify the access transformations that apply to a given class, Maybe `ClassAccessTransformations`?